src/meltano/core/venv_service.py

```diff
@@ -227,7 +227,10 @@ async def create(self) -> Process:
         """
         logger.debug(f"Creating virtual environment for '{self.namespace}/{self.name}'")
         try:
-            return await exec_async(sys.executable, "-m", "venv", str(self.venv))
+            return await exec_async(
+                os.path.join(os.path.dirname(sys.executable), "virtualenv"),
+                str(self.venv),
+            )
```

Comment:
It may be worth exploring the use of `virtualenv` as a Python library rather than a CLI program: https://virtualenv.pypa.io/en/latest/user_guide.html#virtualenv.cli_run

Reply:
@WillDaSilva Yeah, the problem with that is we lose the concurrency of `exec_async`. I might dive a bit in the source of `virtualenv` to see to how hard it'd be to implement an async API for the `Session` object.

Reply:
Good point. We likely won't be able to do it truly concurrently without using multiple processes. With that in mind my guess is that using the CLI will be the best approach in terms of effort versus reward.